internal/state/config/config.go

```diff
@@ -94,6 +96,10 @@ func (cfg *config) SetEndpoint(endpoint string) {
 	cfg.endpoint = endpoint
 }
 
+func (cfg *config) SSHPath() string {
+	return "ssh"
+}
```

Comment:
Maybe this could become a real user preference? And not only be a workaround for mocking.